pjmedia/src/pjmedia/conference.c

```diff
@@ -824,11 +1071,48 @@ PJ_DEF(pj_status_t) pjmedia_conf_create( pj_pool_t *pool_,
     pj_list_init(conf->op_queue);
     pj_list_init(conf->op_queue_free);
 
-    /* Done */
 
+#if defined(PJ_STACK_IMPLEMENTATION)
+    status = pj_stack_create(pool, &conf->unused_slots);
+    if (status != PJ_SUCCESS) {
+        pjmedia_conf_destroy(conf);
+        return status;
+    }
+#else
+    conf->unused_slots = PJ_POOL_ZALLOC_T(pool, unused_slots_cache);
+    pj_list_init(conf->unused_slots);
+#endif
+    conf->free_port_slots = pj_pool_calloc(pool, conf->max_ports, sizeof(port_slot));
+    PJ_ASSERT_ON_FAIL(conf->free_port_slots, {pjmedia_conf_destroy(conf); return PJ_ENOMEM;});
+    unsigned i = conf->max_ports;
```

Comment:
put variable declaration on top of the block, otherwise it may fail to compile on older VS.